CMakeLists.txt

```diff
@@ -1143,13 +1137,9 @@ configure_file("${hpx_SOURCE_DIR}/cmake/templates/hpxrun.py.in"
 # installation instructions
 ################################################################################
 install(
-  DIRECTORY ${CMAKE_BINARY_DIR}/bin/
-  DESTINATION hpx/bin/
-  COMPONENT core
-  FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
-                   GROUP_READ GROUP_EXECUTE
-                   WORLD_READ WORLD_EXECUTE
-  FILES_MATCHING PATTERN "hpxcxx" EXCLUDE)
+  FILES "${CMAKE_BINARY_DIR}/bin/hpxrun.py"
+  DESTINATION bin
+  COMPONENT core)
 
```

Comment:
Heller: look again at the original code. The rule I had removed actually excludes hpxcxx. There are two hpxcxx in the build tree. The original code specifically excluded this hpxcxx as it is configured for the build tree.